fs/accounting/stats_test.go

```diff
@@ -251,6 +251,27 @@ func TestStatsTotalDuration(t *testing.T) {
 	})
 }
 
+func TestRemoteStats(t *testing.T) {
+	ctx := context.Background()
+	startTime := time.Now()
+	time1 := startTime.Add(-40 * time.Second)
+	time2 := time1.Add(10 * time.Second)
+
+	t.Run("Single completed transfer", func(t *testing.T) {
+		s := NewStats(ctx)
+		tr1 := &Transfer{
+			startedAt:   time1,
+			completedAt: time2,
+		}
+		s.AddTransfer(tr1)
+		rs, err := s.RemoteStats()
+
+		require.NoError(t, err)
+		assert.Equal(t, float64(10), rs["transferTime"])
+		assert.Greater(t, rs["elapsedTime"], float64(0))
```

Comment:
This failed on Windows - I guess this is a flaky assertion?

Reply:
The time on Windows is less precise than Linux so this sort of thing is fairly common. I think your 1ms sleep wil fix it though - give that a try.

Reply:
Done - that fixed the test on my Windows machine!